client/my-sites/comments/comment-list/index.jsx

```diff
@@ -214,6 +187,86 @@ export class CommentList extends Component {
 	}
 }
 
+export const CommentFaker = WrappedCommentList => class extends Component {
```

Comment:
@gwwar I had recommended this to @copons as it's supposed to be a dev-only (and small!) piece of code to prevent coupling fake-data operations into the final produce. I'm fine moving it, but I wanted to make note of this fact that it shouldn't be "found" by anyone